frontend/express/public/core/app-management/templates/app-management.html

```diff
@@ -178,7 +178,7 @@ <h2 class="bu-mb-4"> {{apps[selectedApp] ? apps[selectedApp].name : i18n('common
                                 </div>
                                 <p>&nbsp;</p>
                                     <el-upload
-                                    v-if="!isDisabled() && isGlobalAdmin"
+                                    v-if="(!isDisabled() && isGlobalAdmin) || !isGlobalAdmin"
```

Comment:
Better use this:
https://github.com/Countly/countly-server/blob/master/frontend/express/public/core/app-management/javascripts/countly.views.js#L102